docs/pages/modules/module-api.mdx

```diff
@@ -193,6 +193,71 @@ Events("onCameraReady", "onPictureSaved", "onBarCodeScanned")
 
 See [Sending events](#sending-events) to learn how to send events from the native code to JavaScript/TypeScript.
 
+</APIBox>
+<APIBox header="Property">
+
+Defines a value associated with a JavaScript object representing the native module.
+
+To declare a read-only property, you can use a shorthanded syntax that requires only two arguments:
+
+- **name**: `String` — Name of the property that you'll use from JavaScript.
+- **getter**: `() -> ProperyType` — The closure to run when the getter for a property was called.
+
+<CodeBlocksTable>
+
+```swift
+Property("foo") {
+  return "bar"
+}
+```
+
+```kotlin
+Property("foo") {
+  return@Property "bar"
+}
+```
+
+</CodeBlocksTable>
+
+In the case of the mutable property, both the getter and the setter closure are needed (using that syntax is also possible to declare a property with only a setter):
```

Comment:
I'm not sure I understand what is "that syntax" is being referenced in the above line?